### ISSUE TITLE
Investigate issue with article tags

### ISSUE THREAD

**Issue by fatso-jetson** [realworlddatascience/realworlddatascience.github.io#29]
Ideally, clicking on an article tag in a post will take you to that post's main blog index page, with all articles with matching tags displayed.

Currently, however, the tags seem to return users to the last index page, or listing page, from which an article was clicked - a challenge for RWDS, as listings are used throughout the site to highlight a range of content to visitors.

Can this be fixed? Can tags essentially be hard-coded to point to a main listing location - e.g. the latest content index page?

Need to investigate or raise issue with Quarto team.

**Comment by erkanmalcokcom** [comment]
I think this might be a solution.

# Add this
aliases: 
  - /careers/career-profiles/index.html
# To: 
https://github.com/realworlddatascience/realworlddatascience.github.io/blob/main/careers/index.qmd

**Comment by erkanmalcokcom** [comment]
I am still try to understand the website structure. So, if you are in a post, when you click the "statistics" tag, you want to list every post from any blog (Editors, Interviews, etc.). It means you need a custom list. Please tell me if I am wrong.

**Comment by brtarran** [comment]
Yes, that's where I'd like to get to. At the moment, clicking the tags takes you either to the last index/listing page you were on when you clicked the article link, or - for articles accessed directly via a link - users are returned to the homepage.

Ideally, I'd like any article tag to take readers to one main listing page - our latest content page, for example: https://realworlddatascience.net/latest-content.html - regardless of how they access an article, but I'm not sure if there is a way to achieve this in the current version of Quarto.

**Comment by erkanmalcokcom** [comment]
Links are mapped to the location of content in the site maybe in future release.

Here is a workaround solution technically, you need to _redirect_ from 

FROM > https://****/careers/index.html#category=Leadership
TO> https://****/latest-content.html#category=Leadership

I believe that there are alternative non-native solutions, but I am hesitant to consider them because of possible problems with SEO and maintenance.

**Comment by brtarran** [comment]
Thank you. I'll experiment with this, and also raise the tag link issue as a possible feature enhancement to the Quarto team.
